.github/workflows/cmake.yml

```diff
@@ -152,7 +152,18 @@ jobs:
         echo "::group::ninja run # Qemu output"
         ninja run
         echo "::endgroup::"
-      timeout-minutes: 20
+        echo "::group::Verify Output File"
+        mkdir fsmount
+        sudo mount -t ext2 -o loop,rw _disk_image fsmount
+        echo "Results: "
+        sudo cat fsmount/home/anon/test-results.log
+        if ! sudo grep -q "Failed: 0" fsmount/home/anon/test-results.log
+        then
+          echo "::error :^( Tests failed, failing job"
+          exit 1
+        fi
+        echo "::endgroup::"
+      timeout-minutes: 10
```

Comment:
Maybe mention this change in the commit description or something?
seems like the test is timing out though, maybe 10m isn't enough? (I didn't check)

Reply:
Lol it sure did timeout. Sigh. GA with its non homogenous runners. I'll revert that change and update the PR description

Reply:
Updated to 30 minutes instead of going down to 10.